src/migration/1602141176877-migration-V1_0_0.ts

```diff
@@ -0,0 +1,14 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
```

Comment:
if it's generated code why it's not on gitignore?

Reply:
it is required to update db between versions properly if it changes (and for rolling back db version).